### ISSUE TITLE
'Chartboost/Chartboost.h' file not found

### ISSUE THREAD

**Issue by knobapps** [cranberrygame/cordova-plugin-ad-video-chartboost#9]
Exporting using Construct 2, then compiling using cordova, I get the following error in iOS.  I do not get the error in Android.

/Users/user30017/Desktop/BCQuestForLove_iOS/platforms/ios/build/B.C.\ Quest\ for\ Love.build/Debug-iphonesimulator/B.C.\ Quest\ for\ Love.build/Objects-normal/i386/ChartboostPlugin.o
In file included from /Users/user30017/Desktop/BCQuestForLove_iOS/platforms/ios/B.C. Quest for Love/Plugins/cordova-plugin-ad-chartboost/ChartboostPlugin.m:5:
B.C. Quest for Love/Plugins/cordova-plugin-ad-chartboost/ChartboostPlugin.h:6:9: fatal error: 
      'Chartboost/Chartboost.h' file not found
# import <Chartboost/Chartboost.h>

```
    ^
```

1 error generated.

**Comment by knobapps** [comment]
Found you cannot have [.] or spaces in your App name.

**Comment by Ziv-Barber** [comment]
Same here:
'Chartboost/Chartboost.h' file not found
In Android everything is OK.
Guess that we need somehow to install Chartboost SDK but how to do that?

**Comment by knobapps** [comment]
No, I found the fix is to remove any spaces from your app name in the confign.xml
Example
BAD: `<name>My APP</name>`
GOOD: `<name>MyAPP</name>`

**Comment by Ziv-Barber** [comment]
My app name doesn't have any spaces in it.

**Comment by knobapps** [comment]
The other issue I know of, is not using the correct plugin import command line.
Send me an email with what you are typing in and I might be able to help.  I have done this a few times.

**Comment by knobapps** [comment]
support@knobapps.com

**Comment by cranberrygame** [comment]
@knobapps 
@Ziv-Barber 

Try not npm but github url because npm doest not support ios symbolic link.

Use this command:

cordova plugin add cordova plugin add https://github.com/cranberrygame/cordova-plugin-ad-chartboost

Thanks
Sang Ki Kwon

**Comment by Ziv-Barber** [comment]
cordova plugin add https://github.com/cranberrygame/cordova-plugin-ad-chartboost.git worked - thanks!!!

**Comment by cranberrygame** [comment]
You are welcome

**Comment by matrixreal** [comment]
i got same here

**Comment by matrixreal** [comment]
i'm still facing the chartboost.h not found problem even after add it via cordova plugin add https://github.com/cranberrygame/cordova-plugin-ad-chartboost.git
i'm using cordova ios 4.4.0
i did also check my app name and there are no spaces
any help please ?
thank you

**Comment by Ziv-Barber** [comment]
I this issue again with Xcode 9 and Cordova iOS 4.5.1
How can I add the SDK to xcode?

**Comment by Ziv-Barber** [comment]
OK found the solution:

1. Download the Chartboost SDK for iOS from here:

https://answers.chartboost.com/en-us/articles/download

2. Extract it.
3. Copy the Chartboost folder inside it into both plugins/cordova-plugin-ad-chartboost and platforms/ios/your_app_name/plugins/cordova-plugin-ad-chartboost/
4. Close and restart Xcode or it'll not work.
5. Try to compile

**Comment by matrixreal** [comment]
@Ziv-Barber thanks
is cordova ios 4.5.1 good ? faster ?
whats news on it ? 
(i'm using 4.4.0)
